cfme/fixtures/qa_contact.py

```diff
@@ -17,8 +17,8 @@ def dig_code(node):
     cmd_params = ['git', 'blame', line_param, filename, '--show-email']
 
     proc = subprocess.Popen(cmd_params, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
could you simplify it a bit by using wrappers around Popen such as run() instead

Reply:
Done